app/assets/stylesheets/application.scss

```diff
@@ -13,6 +442,14 @@
   color: $black;
 }
 
+.alert {
+  margin-left: 11%;
```

Comment:
Properties should be ordered list-style, margin-bottom, margin-left, margin-right